youtube_dl/extractor/ard.py

```diff
@@ -313,7 +353,7 @@ def _real_extract(self, url):
 
 
 class ARDBetaMediathekIE(ARDMediathekBaseIE):
-    _VALID_URL = r'https://(?:(?:beta|www)\.)?ardmediathek\.de/(?P<client>[^/]+)/(?:player|live|video)/(?P<display_id>(?:[^/]+/)*)(?P<video_id>[a-zA-Z0-9]+)'
+    _VALID_URL = r'https://(?:(?:beta|www)\.)?ardmediathek\.de/(?P<client>[^/]+)/(?P<mode>player|live|video|sendung|sammlung)/(?P<display_id>(?:[^/]+/)*)(?P<video_id>[a-zA-Z0-9]+)'
```

Comment:
_> It must be a separate extractor in the first place._

Do I understand you correctly?
- I leave `class ARDBetaMediathekIE(ARDMediathekBaseIE)` unchanged (so that it's only responsible for single videos).
- And instead, I create a new class for Playlists within `ard.py`:
```
	class ARDMediathekPlaylistIE(ARDMediathekBaseIE):
		_VALID_URL = r'https://(?:beta|www)\.ardmediathek\.de/...'
		... 
```

- And I add the new playlist class to `extractors.py`:
`	from .ard import ( ... ARDMediathekPlaylistIE )
`

I searched a lot to understand the Playlist stuff in youtube-dl, but I could not find coding conventions for this topic. Is there something like a playlist-best-practice-guide?